.cicd/release.sh

```diff
@@ -15,4 +15,14 @@ set -eou pipefail
   helm package .
 
   mv "weaviate-$VERSION.tgz" "weaviate.tgz"
+
+  echo "Add chart version $VERSION to GitHub Pages"
+  cd ..
+  mkdir -p files-to-gh-pages
+  echo $(pwd)
```

Comment:
Can this line be removed?

Reply:
Same as `echo`, I think this is to make sure everything is installed where it is supposed to. I do not think it is used besides debugging.

Reply:
ok, let's leave it then